### PR TITLE
164925035 After translate language, user should see the change of translation

### PR DESCRIPTION
**Pull request by oNguyenNgocTrung**
[#164925035](https://www.pivotaltracker.com/story/show/164925035)

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -20,6 +20,6 @@
 
   # match '/users/finish_signup' => 'users/registrations#finish_signup', via: [:get, :patch], :as => :finish_signup
 
-  ActiveAdmin.routes(self)
+  ActiveAdmin.routes(self) rescue ActiveAdmin::DatabaseHitDuringLoad
```
Comment (oNguyenNgocTrung) [inline config/routes.rb]: Fix error `ActiveAdmin::DatabaseHitDuringLoad` when run "rake db:create"

File: config/initializers/locale.rb [hunk 2]
```diff
@@ -10,11 +10,26 @@
...
+
+module I18n
+  module Backend
+    class ActiveRecord
+      class Translation < ::ActiveRecord::Base
+        after_save :reload_i18n
+
+        private
+        def reload_i18n
+          I18n.backend.reload!
```
Comment (tri-dang) [inline config/initializers/locale.rb]: nice monkey patching here

File: config/routes.rb (hunk 1 above)
Comment (tri-dang) [reply config/routes.rb]: can you tell more about the case leads to this error?

Comment (oNguyenNgocTrung) [reply config/routes.rb]: when calling: `ActiveAdmin.routes(self)` it then opens `app/admin/#{model}.rb` which attempts to connect to the DB and raises an error if DB don't exist yet.
ảnh

**Pull request merged**